config/routes.rb

```diff
@@ -1,4 +1,4 @@
-Rails.application.routes.draw do
+Rails.application.routes.draw do # rubocop:disable Metrics/BlockLength
```

Comment:
A better alternative to this might be to do something like https://github.com/alphagov/collections-publisher/commit/5579586fee81eda19639bbde2dc21f0002f16806 but for `routes.rb`.

Reply:
That makes it fail with the same cop on `spec/models/healthcheck/subscription_content_healthcheck.rb` for some reason.  If I also exclude that file, a bunch more `spec` files start to fail.

Reply:
Oh... that's weird. Fine with this comment then 👍